### DISCUSSION TITLE
How to crop page from input PDF file and export to rasterized image in PDF file

### DISCUSSION THREAD

**Discussion by MisiekDP** [ImageMagick/ImageMagick#5733] [Help] [answered]
Hey,
how to crop a PDF file with white borders to other PDF?

I mean to convert and cut PDF file:
![!ONEinput](https://user-images.githubusercontent.com/22171619/200203812-a3acdb2b-c86b-4256-8d66-0ba520b733b7.jpg)

To this file:
![!ONEinput-OUT2IMG](https://user-images.githubusercontent.com/22171619/200203886-bf9f5c8e-87df-42f4-95ac-8677302acb52.jpg)

But the input and output file must be a in PDF file format. I tried a `-trim` command but it works with only JPEG files (when I tested).

And later of this I need to add a white border from all four corners about 5% of horizontal size of rasterized image.

The original PDF file:
[!ONEinput.pdf](https://github.com/ImageMagick/ImageMagick/files/9947206/ONEinput.pdf)

**Comment by snibgo** [comment]
How about:
```
magick ONEinput.pdf -trim +repage out.pdf
```
This should work, provided IM can find Ghostscript. The output file will contain a raster image. For better quality, use a "-density" setting before reading the input.

**Reply by MisiekDP** [reply]
Yea! It works 🗡️ 
Btw. how could I add these white borders around the picture?

**Comment by snibgo** [comment, marked as answer]
To add a 10-pixel border:
```
magick ONEinput.pdf -trim +repage -bordercolor White -border 10 out.pdf
```
Instead of "10", we can use an expression:
```
magick ONEinput.pdf -trim +repage -bordercolor White -border %[fx:w*0.05] out.pdf
```
